ldes-server-compaction/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/compaction/domain/entities/CompactedFragment.java

```diff
@@ -0,0 +1,80 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.compaction.domain.entities;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.model.FragmentPair;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.model.LdesFragmentIdentifier;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.model.TreeRelation;
+import be.vlaanderen.informatievlaanderen.ldes.server.fetching.entities.CompactionCandidate;
+import be.vlaanderen.informatievlaanderen.ldes.server.fragmentation.entities.Fragment;
+import be.vlaanderen.informatievlaanderen.ldes.server.fragmentation.exceptions.MissingFragmentValueException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static be.vlaanderen.informatievlaanderen.ldes.server.fragmentation.services.FragmentComparator.sortFragments;
+
+public class CompactedFragment {
+	public static final String PAGE_NUMBER_KEY = "pageNumber";
+	private final Set<CompactionCandidate> toBeCompactedFragments;
+
+	public CompactedFragment(Set<CompactionCandidate> toBeCompactedFragments) {
+		this.toBeCompactedFragments = toBeCompactedFragments;
+	}
+
+	public Fragment getFragment() {
+		return new Fragment(getLdesFragmentIdentifier(), true,
+				getFirstImpactedFragment().getNrOfMembersAdded(), getOutgoingRelations(), null);
```

Comment:
Why do we use the nrOfMembersAdded from the first fragment? Shouldn't this be the sum of the sizes of the CompactionCandidates?